core/touch.js

```diff
@@ -107,6 +107,9 @@ Blockly.onMouseUp_ = function(e) {
     return;
   }
   Blockly.Touch.clearTouchIdentifier();
+  // Check whether we should do this regardless of drag mode.  See comment on
```

Comment:
Make this // TODO: (issue number).  Right now it looks like it's saying that the code is doing this check.